src/libgetenv/src/getenv.cpp

```diff
@@ -98,7 +98,11 @@ KeySet *elektraDocu = ksNew(20,
 #include "readme_elektrify-getenv.c"
 	KS_END);
 
-pthread_mutex_t elektraGetEnvMutex = PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP;
+#if defined(__APPLE__) && defined(__MACH__)
+	pthread_mutex_t elektraGetEnvMutex = PTHREAD_RECURSIVE_MUTEX_INITIALIZER;
```

Comment:
it appears it's available on OS X (not tested extensively yet), don't know about other BSDs